install/README.md

```diff
@@ -207,22 +268,24 @@ Refer to the [HPE Cray EX System Software Getting Started Guide S-8000](https://
 on the HPE Customer Support Center for more information on other product streams
 to be installed and configured after CSM.
 
-### Troubleshooting installation problems
+### 12. Troubleshooting installation problems
```

Comment:
We had left this as unnumbered because it isn't an actual step, it's an old-ball header section. It wouldn't hurt to number it, but I wanted to share the backstory.